web/tailwind.config.js

```diff
@@ -5,7 +5,34 @@ module.exports = {
     preflight: false, // NOTE: remove when ready to implement styles across browser
   },
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        yote: {
+          50: '#FBA640',
+          100: '#FB9741',
+          200: '#FA8A41',
+          300: '#FB7C40',
+          400: '#FA703F',
+          500: '#FB6440',
+          600: '#FB4840',
+          700: '#ED393B',
+          800: '#DF2B39',
+          900: '#D41136',
+        },
+        green: {
+          50: '#E7FFF2',
+          100: '#BEFFDD',
+          200: '#90FFC5',
+          300: '#62FFAD',
+          400: '#11FF83',
+          500: '#0DF07A',
+          600: '#0AD46B',
+          700: '#07A452',
+          800: '#057339',
+          900: '#005127',
+        }
+      }
```

Comment:
Can you add a brief comment here stating what this does?

Reply:
Going to pull these specific colors out, as they were more for illustrating an example in my post. Outside of example, for which Tailwind docs should be the source anyways, I don't see a case for including these in a generic starting point.